security/permissions.rst

```diff
@@ -0,0 +1,103 @@
+
+Permissions (AccessPolicy)
+--------------------------
+Firely Server supports filtering the requested client scopes by using built-in custom access policy resources. Both Smart-V1 and Smart-V2 scopes are supported. 
+The resource definitions are preloaded in Firely Server and can be viewed on the administration endpoint API or in Simplier under 
```

Comment:
```suggestion
The StructureDefinitions are preloaded in Firely Server and can be viewed on the administration endpoint API or in Simplifier under 
```

Reply:
.suo file was added in the last commit. Consider adjusting .gitignore or removing the file